docs/tutorials/orc.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<p>This section can be renamed to "dataset pre-processing".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>This section is model building. i can add a subsection for data preprocessing.</p>

Reply:
<p>This section title can be removed. Please check the comments above and below.</p>